src/app/debug/table/table.component.ts

```diff
@@ -478,10 +525,14 @@ export class TableComponent implements OnInit, OnDestroy {
   }
 
   openReport(storageId: string): void {
-    this.httpService.getReport(storageId, this.viewSettings.currentView.storageName).subscribe((data: Report): void => {
-      data.storageName = this.viewSettings.currentView.storageName;
-      this.openReportEvent.next(data);
-    });
+    if (storageId && this.viewSettings.currentView?.storageName) {
```

Comment:
`storageId` has type string so it can never be null here, so this if statement actually reads: 
```ts
    if (true && this.viewSettings.currentView?.storageName) {
```

Reply:
You're totally right, i got rid of it